protocols/raft/src/main/java/io/atomix/protocols/raft/roles/LeaderRole.java

```diff
@@ -1403,7 +1404,7 @@ private void commitCommand(CommandRequest request, CompletableFuture<CommandResp
                   log.trace("Appended {}", indexed);
                   return indexed;
                 });
-      } catch (StorageException.TooLarge e) {
+      } catch (StorageException.TooLarge | BufferOverflowException e) {
```

Comment:
why?

Reply:
It can be thrown if you're trying to write an entry larger than the total segment, see https://github.com/zeebe-io/atomix/blob/4a65503db2c810df7aa9f9b1dc0aec4debe21380/storage/src/main/java/io/atomix/storage/journal/FileChannelJournalSegmentWriter.java#L246

Reply:
:ok_hand: